---
source: crates/zizmor/tests/integration/snapshot.rs
expression: "zizmor().expects_failure(true).offline(true).unsetenv(\"GH_TOKEN\").args([\"pypa/sampleproject\"]).run()?"
---
fatal: no audit was performed
error: can't retrieve repository: pypa/sampleproject
  |
  = help: try removing --offline or passing --gh-token <TOKEN>
